GitUI/UserControls/ExListView.cs

```diff
@@ -55,7 +55,7 @@ public enum ListViewGroupState : uint
         SubSetLinkFocused = 128
     }
 
-    internal class ExListView : NativeListView
+    internal class ExListView : ListView
```

Comment:
I'm not convinced this is a good fix as this essentially rolls back #4491

Reply:
I managed to fix it without breaking NativeListView selection rendering